Ctrl crash

soloturn opened the pull request:
to better find the cause of  #5224 , crash on right ctrl, it catches all exceptions. the error on right-ctrl is then when debugging in intellij:
```
Caused by: java.lang.NullPointerException: Cannot load from byte/boolean array because "this.deflated" is null

```

when running with `gradle game` : 
```
java.lang.IllegalArgumentException: moduleName must not be null or empty
        at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
        at org.terasology.gestalt.assets.ResourceUrn.<init>(ResourceUrn.java:185)
        at org.terasology.gestalt.assets.ResourceUrn.<init>(ResourceUrn.java:128)
        at org.terasology.engine.rendering.nui.internal.NUIManagerInternal.bindEvent(NUIManagerInternal.java:770)
        at org.terasology.engine.rendering.nui.internal.NUIManagerInternalMethodAccess.invoke(Unknown Source)
        at org.terasology.engine.entitySystem.event.internal.EventSystemImpl$ByteCodeEventHandlerInfo.invoke(EventSystemImpl.java:399)
        at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.sendConsumableEvent(EventSystemImpl.java:274)
        at org.terasology.engine.entitySystem.event.internal.EventSystemImpl.send(EventSystemImpl.java:253)
        at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
        at org.terasology.engine.network.NetworkEventSystemDecorator.send(NetworkEventSystemDecorator.java:54)
        at org.terasology.engine.core.bootstrap.eventSystem.AbstractEventSystemDecorator.send(AbstractEventSystemDecorator.java:67)
        at org.terasology.engine.recording.RecordingEventSystemDecorator.send(RecordingEventSystemDecorator.java:32)
        at org.terasology.engine.entitySystem.entity.internal.BaseEntityRef.send(BaseEntityRef.java:188)
        at org.terasology.engine.input.internal.BindableButtonImpl.updateBindState(BindableButtonImpl.java:153)
        at org.terasology.engine.input.InputSystem.updateBindState(InputSystem.java:314)
        at org.terasology.engine.input.InputSystem.processKeyboardInput(InputSystem.java:389)
        at org.terasology.engine.input.InputSystem.update(InputSystem.java:132)
```
but it stays alive.

soloturn committed:
do not exist for unexpected exceptions

soloturn committed:
do not exist for unexpected exceptions

Cervator commented:
Rebased this one as a relatively recent and small PR just to validate infra changes (to see if this PR passes all the checks)

Hoping to finally get back into some real review and coding when infra work is finally over, hopefully soon :-)

soloturn commented:
> **AI-assisted update.** Filed by agent driven by @soloturn via [GDD](https://github.com/soloturn/yggdrasil).

Superseded. This wrapped `tick()` in a catch-all to keep the game alive long enough to observe the two crashes described here, tied to #5224. Both are now properly root-caused and fixed rather than swallowed:

- `"this.deflated is null"` - the light/mesh-worker data race, fixed in #5374 (`ChunkLightLocks`).
- `"moduleName must not be null or empty"` in `NUIManagerInternal.bindEvent` - `SortOrderSystem` building fallback bind ids with the wrong `SimpleUri` constructor, fixed in #5376. #5224 is already closed referencing that fix.

Closing this in favor of those two.
